tests/sdk/flagsmith-identity-flags.test.ts

```diff
@@ -155,3 +155,31 @@ test('test_get_identity_flags_multivariate_value_with_local_evaluation_enabled',
   expect(identityFlags.getFeatureValue('mv_feature')).toBe('bar');
   expect(identityFlags.isFeatureEnabled('mv_feature')).toBe(false);
 });
+
+
+test('test_transient_identity', async () => {
+  // @ts-ignore
+  fetch.mockReturnValue(Promise.resolve(new Response(traitIdentityJSON())));
+  const identifier = 'transient_identifier';
+  const traits = { some_trait: 'some_value' };
+  const transient = true;
+  const flg = flagsmith();
+  const identityFlags = (await flg.getIdentityFlags(identifier, traits, transient)).allFlags();
+  expect(identityFlags[0].enabled).toBe(false);
+  expect(identityFlags[0].value).toBe('some-transient-identity-value');
+  expect(identityFlags[0].featureName).toBe('some_feature');
+});
+
+
+test('test_identity_with_transient_traits', async () => {
+  // @ts-ignore
+  fetch.mockReturnValue(Promise.resolve(new Response(identityWithTransientTraitsJSON())));
+  const identifier = 'transient_trait_identifier';
+  const traits = { some_trait: 'some_value', another_trait: {value: 'another_value', transient: true} };
+  const flg = flagsmith();
+
+  const identityFlags = (await flg.getIdentityFlags(identifier, traits)).allFlags();
+  expect(identityFlags[0].enabled).toBe(true);
+  expect(identityFlags[0].value).toBe('some-identity-with-transient-trait-value');
+  expect(identityFlags[0].featureName).toBe('some_feature');
+});
```

Comment:
These tests, essentially, do nothing. They mock responses and validate the mocks.

I expect them to validate the request bodies instead. In fact, if they did it, the problems I highlighted above would be caught by them.

Reply:
Corrected